lib/src/supabase.dart

```diff
@@ -39,15 +39,34 @@ class Supabase {
   ///
   /// This must be called only once. If called more than once, an
   /// [AssertionError] is thrown
+  ///
+  /// [url] and [anonKey] can be found on your Supabase dashboard.
+  ///
+  /// You can access none public schema by passing different [schema].
+  ///
+  /// Default headers can be overridden by specifying [headers].
+  ///
+  /// Specify [authCallbackUrlHostname] to let the SDK know what host name auth callback deeplink will have.
+  /// if [authCallbackUrlHostname] is not set, we treat all deeplinks as auth callbacks.
+  ///
+  /// Pass [localStorage] to override the default local storage option used to persist auth.
+  ///
+  /// Custom http client can be used by passing [httpClient] parameter.
+  ///
+  /// [storageRetryAttempts] specifies how many retry attempts there should be to
+  ///  upload a file to Supabase storage when failed due to network interruption.
+  ///
+  /// If [debug] is set to `true`, debug logs will be printed in debug console.
   static Future<Supabase> initialize({
     required String url,
     required String anonKey,
+    String? schema,
+    Map<String, String>? headers,
     String? authCallbackUrlHostname,
-    bool? debug,
     LocalStorage? localStorage,
     Client? httpClient,
-    Map<String, String>? headers,
-    String? schema,
+    int storageRetryAttempts = 0,
+    bool debug = kDebugMode,
```

Comment:
This could be a breaking change

Reply:
Why? `headers` and `schema` were moved to the top of the parameters list

Reply:
I meant the `debug` parameter. Previously you could assign `null`, but now it has to be non null bool.

Reply:
@Vinzent03 Reverted!